nemo/collections/nlp/models/language_modeling/megatron_bert_model.py

```diff
@@ -250,7 +250,7 @@ def loss_func(output_tensor):
                     lm_loss = loss_dict['lm loss']
                     loss = lm_loss
                     reduced_loss = average_losses_across_data_parallel_group([loss, lm_loss])
-                return loss, {'avg': reduced_loss}
+                return loss, {'loss': reduced_loss}
```

Comment:
Why are we changing 'avg' to 'loss' ?

Reply:
`avg` is not informative. It implicitly relates to `avg_loss` but since I can return multiple items with hidden transformations (all are averaged) I made it explicit. It does deviate from the GPT convention which returns `avg`. 
If you would like to have a unified convention I would recommend that I will also update the GPT models to return the same key.
Does that make sense?

Reply:
Kept as is, as discussed.